apps/web/components/PayPal/GooglePayButton.vue

```diff
@@ -0,0 +1,212 @@
+<template>
+  <div class="flex items-center justify-center w-full my-2" v-if="isGooglePayLoaded">
+    <div class="border-t-2 flex-grow"></div>
+    <p class="px-2 text-sm uppercase text-gray-400">{{ $t('or') }}</p>
+    <div class="border-t-2 flex-grow"></div>
+  </div>
+  <div id="google-pay-button"></div>
+</template>
+
+<script lang="ts" setup>
+import { PayPalAddToCartCallback } from '~/components/PayPal/types';
+import { cartGetters, orderGetters } from '@plentymarkets/shop-api';
+
+let isGooglePayLoaded = true;
+let countryCodeString = 'DE';
```

Comment:
isGooglePayLoaded should be `false` by default and a ref().
Country code is hardcoded that should be fixed as well.

Reply:
We couldn't find any device which doesn't accept Googlepay, that's why we set it always true